ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -301,6 +455,7 @@ export default function TransactionList({
       }
       <Box className="transaction-list" {...boxProps}>
         <Box className="transaction-list__transactions">
+          {renderFilterButton()}
```

Comment:
This should be behind a feature flag for now.

Reply:
@gambinish used `FILTER_TOKENS_TOGGLE` in his PR but I think we should just go with a flag called `PORTFOLIO_VIEW`